src/main/sensors/gyro.h

```diff
@@ -64,20 +64,27 @@ typedef struct gyroConfig_s {
     uint8_t  gyroMovementCalibrationThreshold; // people keep forgetting that moving model while init results in wrong gyro offsets. and then they never reset gyro. so this is now on by default.
     uint8_t  gyro_sync_denom;                  // Gyro sample divider
     uint8_t  gyro_lpf;                         // gyro LPF setting - values are driver specific, in case of invalid number, a reasonable default ~30-40HZ is chosen.
-    uint8_t  gyro_soft_lpf_type;
-    uint8_t  gyro_soft_lpf_hz;
+
+    // "Stage 2" is the older soft LPF: Biquad, PT1 or Denoise. Applied after dynamic notch and static notch 1/2.
+    uint8_t  gyro_stage2_soft_lpf_type;
+    uint8_t  gyro_stage2_soft_lpf_hz;
+
     bool     gyro_high_fsr;
     bool     gyro_use_32khz;
     uint8_t  gyro_to_use;
-    uint16_t gyro_soft_lpf_hz_2;
+
+    // "Stage 1" is the FKF, Biquad RC+FIR2, or PT1 options. Applied before dynamic notch.
+    uint8_t  gyro_stage1_soft_lpf_type;
+    uint16_t gyro_stage1_soft_lpf_hz;
+
+    uint8_t gyro_lma_depth;
```

Comment:
I'll move these up above the stage1 so all the uint8_t's are in a row (no struct filler)